### PR TITLE
Release 1.19.0-beta.0

### PR DESCRIPTION
**Pull request by ilya-zuyev**
Release v1.19.0-beta.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,132 @@
...
+* Addon: bump csi-hostpath-driver to v1.6.0 [#10798](https://github.com/kubernetes/minikube/pull/10798)
+* Bump github.com/otiai10/copy from 1.0.2 to 1.5.0 [#10891](https://github.com/kubernetes/minikube/pull/10891)
+* Bump github.com/shirou/gopsutil/v3 from 3.21.1 to 3.21.2 [#10893](https://github.com/kubernetes/minikube/pull/10893)
+* Upgrade ingress addon files according to upstream(ingress-nginx v0.44.0) [#10879](https://github.com/kubernetes/minikube/pull/10879)
+* Bump k8s.io/klog/v2 from 2.4.0 to 2.8.0 [#10894](https://github.com/kubernetes/minikube/pull/10894)
+* Bump k8s.io/client-go from 0.18.8 to 0.20.5 [#10890](https://github.com/kubernetes/minikube/pull/10890)
+* Bump github.com/cheggaaa/pb/v3 from 3.0.1 to 3.0.6 [#10828](https://github.com/kubernetes/minikube/pull/10828)
+* Bump golang.org/x/mod from 0.4.1 to 0.4.2 [#10829](https://github.com/kubernetes/minikube/pull/10829)
+* Bump github.com/google/go-cmp from 0.5.4 to 0.5.5 [#10747](https://github.com/kubernetes/minikube/pull/10747)
+* Update go-container registry to 0.4.1 + patch [#10731](https://github.com/kubernetes/minikube/pull/10731)
```
Comment (medyagh) [inline CHANGELOG.md]: for Version changes, lets remove all the Go Mod version changes

only version changes that affect user will need to be included

for example Crictl version or Docker Version or Pdoman version

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,132 @@
...
+* Correct spelling in --insecure-registry validation error message [#10735](https://github.com/kubernetes/minikube/pull/10735)
+* Add flag "--listen-address" for docker and podman driver [#10653](https://github.com/kubernetes/minikube/pull/10653)
+* kvm: provide solution if user doesn't belong to libvirt group [#10712](https://github.com/kubernetes/minikube/pull/10712)
+* CoreDNS early scale down to 1 replica [#10656](https://github.com/kubernetes/minikube/pull/10656)
+* Wait for crictl version after the socket is up [#10705](https://github.com/kubernetes/minikube/pull/10705)
+
+Bug Fixes:
+
+* Fix CNI issue related to picking up wrong CNI   [#10985](https://github.com/kubernetes/minikube/pull/10985)
+* Added error check for extra quotation for extra-config. [#10886](https://github.com/kubernetes/minikube/pull/10886)
```
Comment (medyagh) [inline CHANGELOG.md:60]: extra validation for extra-config

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,132 @@
...
+* Fix the failure of `minikube mount` in case of KVM2 [#10733](https://github.com/kubernetes/minikube/pull/10733)
+* Fix/minikube status for scheduled stop [#10911](https://github.com/kubernetes/minikube/pull/10911)
+* create network: use locks and reservations to solve race condition [#10858](https://github.com/kubernetes/minikube/pull/10858)
+* fix driver.IndexFromMachineName() [#10821](https://github.com/kubernetes/minikube/pull/10821)
+* Fix link for pushing images docs on cache cmd [#10880](https://github.com/kubernetes/minikube/pull/10880)
```
Comment (medyagh) [inline CHANGELOG.md:65]: remove  doc related changes

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,132 @@
...
+* Enable portmap for the default cni bridge [#10782](https://github.com/kubernetes/minikube/pull/10782)
+* install losetup from util-linux in the ISO to enable support for VolumeMode=Block PVCs [#10704](https://github.com/kubernetes/minikube/pull/10704)
+* auto-detect gce and do not enable gcp auth addon [#10730](https://github.com/kubernetes/minikube/pull/10730)
+* add validations --image-repository inputs [#10760](https://github.com/kubernetes/minikube/pull/10760)
+* docker-env & podman-env: silent output when talking to a shell [#10763](https://github.com/kubernetes/minikube/pull/10763)
+* The cluster doesn't have to be healthy for kubectl [#10732](https://github.com/kubernetes/minikube/pull/10732)
+* Need to exit if unable to cache kubectl [#10734](https://github.com/kubernetes/minikube/pull/10734)
+* increase wait for docker starting on windows [#10765](https://github.com/kubernetes/minikube/pull/10765)
+* Correct spelling in --insecure-registry validation error message [#10735](https://github.com/kubernetes/minikube/pull/10735)
+* Add flag "--listen-address" for docker and podman driver [#10653](https://github.com/kubernetes/minikube/pull/10653)
```
Comment (medyagh) [inline CHANGELOG.md:52]: add this to Features

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,132 @@
 # Release Notes
 
+## Version 1.19.0-beta.0 - 2021-04-05
+
+Features:
+
+* iso: enable Network Block Device support [#10217](https://github.com/kubernetes/minikube/pull/10217)
```
Comment (medyagh) [inline CHANGELOG.md:7]: this could be moved to the last line of features, more important features at top

**Review by medyagh — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 4 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 3 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 1 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,112 @@
 # Release Notes
 
+## Version 1.19.0-beta.0 - 2021-04-05
+
+Features:
+
+* add minikube image rm command [#10924](https://github.com/kubernetes/minikube/pull/10924)
```
Comment (medyagh) [inline CHANGELOG.md:7]: lets add single quote for any command, it show it gray
```
add `minikube image rm` command
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,112 @@
...
+* create registry credentials when gcp-auth addon is enabled [#10853](https://github.com/kubernetes/minikube/pull/10853)
```
Comment (medyagh) [inline CHANGELOG.md:8]: lets add colons liek this

GCP-Auth addon: Add support for GCR creds

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,112 @@
...
+* new command: `minikube cp` to copy files into minikube [#10198](https://github.com/kubernetes/minikube/pull/10198)
+* Add flag "--listen-address" for docker and podman driver [#10653](https://github.com/kubernetes/minikube/pull/10653)
```
Comment (medyagh) [inline CHANGELOG.md:10]: consistent woring,

since previous one is "new" we can say
new flag ...

**Review by medyagh — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 7 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 6 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 8 above)
Comment (ilya-zuyev) [reply CHANGELOG.md]: done

**Pull request merged**